### PR TITLE
Update Starting Latitude, Lower Longitude, and README Documentation

### PR DESCRIPTION
**Pull request by ezhilsabareesh8**
This PR updates the starting latitude and lower longitude parameters in the ocean grid generator. Additionally, the README has been updated to implement [these required changes](https://github.com/COSIMA/access-om3/issues/172#issuecomment-2232235022) in the new grid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ To test this module quickly try
 - cd extras ; make quick
 
 Examples:
-- To build a grid consisting a 1/2 degree regular spherical grid between 50S and 70S, stitched with  a displaced pole cap south of 70S with the south pole displaced to (60E,50S):
+- To build a grid with a 1/4 degree resolution that omits the southern cap, ensures an even number of latitude points, and creates a bipolar grid below 65°N. The grid transitions to a Mercator projection from 75°S to 65°N, matching the grid spacing to that of the Southern Ocean. Additionally, the grid is shifted to align the equator with a u-point:
```
Comment (anton-seaice) [inline README.md:11]:
```suggestion
- To build a grid with a 1/4 degree nominal resolution that omits the southern cap, ensures an even number of latitude points, and creates a bipolar grid north of 65°N. The grid transitions to a Mercator projection from 75°S to 65°N, matching the grid spacing to that of the Southern Ocean. Additionally, the grid is shifted to align the equator with a u-point:
```

File: ocean_grid_generator.py [hunk 2]
```diff
@@ -1077,7 +1077,7 @@ def main(
     ###
     ###Southern Ocean grid
     ###
-    lat0_SO = -78.0  # Starting lower lat of Southern Ocean grid
+    lat0_SO = -81.0  # Starting lower lat of Southern Ocean grid
     if south_ocean_lower_lat > -90:
         lat0_SO = south_ocean_lower_lat
```
Comment (anton-seaice) [inline ocean_grid_generator.py:1082]: Can we not use the `southern_ocean_lower_lat` argument instead of changing this ?

File: README.md
Comment (anton-seaice) [inline README.md:1]: I guess we could add instructions to install this from pip? @dougiesquire Shall we add this package to pip ?

**Review by anton-seaice — COMMENTED** [review]
Technically we should update this line too:

https://github.com/ACCESS-NRI/ocean_model_grid_generator/blob/8465313bb73a203d51e2d86849be0656b2bcefd4/ocean_grid_generator.py#L961

Can you add a comment with the metadata that this includes in the output file ?

**Comment by ezhilsabareesh8** [comment]
> Technically we should update this line too:
> 
> https://github.com/ACCESS-NRI/ocean_model_grid_generator/blob/8465313bb73a203d51e2d86849be0656b2bcefd4/ocean_grid_generator.py#L961
> 
> Can you add a comment with the metadata that this includes in the output file ?

This line is already added, refer [here](https://github.com/ACCESS-NRI/ocean_model_grid_generator/commit/ccd61f3ee6582fce40b7e668e30b70798cf3124a)

File: ocean_grid_generator.py (hunk 2 above)
Comment (ezhilsabareesh8) [reply ocean_grid_generator.py]: Updated in this commit d9799fd

File: ocean_grid_generator.py [hunk 3]
```diff
@@ -969,7 +969,7 @@ def main(
     refineS = 2  # factor 2 is for supergrid
     refineR = degree_resolution_inverse
     lenlon = 360  # global longitude range
-    lon0 = -300.0  # Starting longitude of the map
+    lon0 = -280.0  # Starting longitude of the map
```
[the remark this answers is not in the document]
Comment (ezhilsabareesh8) [reply ocean_grid_generator.py:972]: All our grids start at -280 degrees longitude. We can make this a command-line option if needed.

[the remark this answers is not in the document]
Comment (anton-seaice) [reply ocean_grid_generator.py:972]: I still think we should just do this now - its easy enough to do and makes it clearer:

e.g. add a command line argument `--lower-lon` and then make it an argument to main()

[the remark this answers is not in the document]
Comment (ezhilsabareesh8) [reply ocean_grid_generator.py]: Added in this commit [4ad854e](https://github.com/ACCESS-NRI/ocean_model_grid_generator/pull/1/commits/4ad854e15e13f3631ebffd77b45d2fdbd1c3958c)

**Review by anton-seaice — APPROVED** [review]
Looks good, thanks

**Pull request merged**